Source/diablo.cpp

```diff
@@ -168,6 +169,17 @@ void run_game_loop(unsigned int uMsg)
 	force_redraw = 255;
 	gbGameLoopStartup = TRUE;
 	nthread_ignore_mutex(FALSE);
+	char speed_str[32];
+
+	if (!getIniValue("devilutionx", "game speed", speed_str, sizeof(speed_str))) {
+
+		game_speed = 50;
+	}
+	else {

```

Comment:
Minor nit-pick. Put these on the same line.

```diff
-}
-else {
+} else {
```

This is the convention used in the rest of the codebase:

```
$ ack -x cpp "} else {" devilutionX | wc -l
843
```

Reply:
also the empty lines make my eyes bleed and pants wither! :)